vimish-fold.el

```diff
@@ -131,6 +131,15 @@ overhead is undesirable."
   :type 'boolean
   :package-version '(vimish-fold . "0.2.3"))
 
+(defcustom vimish-fold-allow-nested nil
+  "Whether to allow nested folds
+
+If set to nil, the nested folds (having a fold over a fold) are not allowed
+and will not be created."
+  :tag "Allow nested folds."
+  :type 'boolean
+  :package-version '(vimish-fold . "0.2.3"))
```

Comment:
Let's have 0.2.4 here, because I'll release a new version with this feature.

Reply:
Done